hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java

```diff
@@ -692,11 +694,12 @@ public S3AInputPolicy getInputPolicy() {
   }
 
   /**
-   * Get the change detection policy for this FS instance.
+   * Get the change detection policy for this FS instance.  Only public to allow
+   * access in tests in other packages.
```

Comment:
That's what @VisibleIsTesting is for.

Reply:
Eh... still not a bad thing to remind people of though. +1.

Reply:
Yeah, many are used to @VisibleForTesting on protected methods but less so on public ones.  Given this is public I figured worth calling out in the javadoc as well.  There was a bit of [discussion](https://github.com/apache/hadoop/pull/646#discussion_r269729362) about it in the previous review.